From 4f584b38220e2ba02a039b596db4898b74b05f31 Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Thu, 5 Oct 2017 14:49:09 +0200
Subject: [PATCH 2/2] lua-factory: Fix warning in Apple trailers source

We should make the filesize a number, not pass it
down as a string.

[lua-library] grl-lua-library.c:399: 'string' is not compatible for 'size'
---
 src/lua-factory/sources/grl-appletrailers.lua | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lua-factory/sources/grl-appletrailers.lua b/src/lua-factory/sources/grl-appletrailers.lua
index 03ae1f7..4330c2f 100644
--- a/src/lua-factory/sources/grl-appletrailers.lua
+++ b/src/lua-factory/sources/grl-appletrailers.lua
@@ -120,7 +120,7 @@ function parse_results(results)
     media.title = item.info.title.xml
     media.thumbnail = item.poster.xlarge.xml
     media.url = item.preview.large.xml
-    media.size = item.preview.large.filesize
+    media.size = tonumber(item.preview.large.filesize)
     local mins, secs = item.info.runtime.xml:match('(%d):(%d)')
     media.duration = tonumber(mins) * 60 + tonumber(secs)
 
-- 
2.14.1

